plugins/modules/mysql_user.py

```diff
@@ -45,7 +45,7 @@
     default: false
   priv:
     description:
-      - "MySQL privileges string in the format: C(db.table:priv1,priv2)."
+      - "MySQL privileges string in the format: C(db.table:priv1,priv2). Additionally, there must be no spaces between the table and the privilege as this will yield a non-idempotent check mode."
```

Comment:
```suggestion
      - "MySQL privileges string in the format: C(db.table:priv1,priv2)."
      - Additionally, there must be no spaces between the table and the privilege as this will yield a non-idempotent check mode.
```
this should fix the sanity errors

Reply:
just press "Commit suggestion"